webpack.config.externals.js

```diff
@@ -18,39 +19,27 @@ module.exports = {
             {
                 test: /\.js$/,
                 exclude: /(node_modules|mathquill)/,
-                use: {
-                    loader: "babel-loader",
-                },
+                use: ["babel-loader"],
             },
             {
                 test: /\.less$/,
-                use: [
-                    {
-                        loader: 'css-loader',
-                    },
-                    {
-                        loader: 'less-loader',
-                    },
-                ],
+                use: [MiniCssExtractPlugin.loader, 'css-loader', 'less-loader'],
             },
             {
                 test: /\.css$/,
-                use: [
-                    {
-                        loader: 'css-loader',
-                    }
-                ],
+                use: [MiniCssExtractPlugin.loader, 'css-loader'],
             },
             {
                 test: /\.(woff|woff2|ttf|otf|eot|svg)$/,
-                use: [
-                    {
-                        loader: "file-loader",
-                    },
-                ],
+                use: ["file-loader"],
             }
         ],
     },
+    plugins: [
+        new MiniCssExtractPlugin({
+            filename: "math-input.css",
+        }),
+    ],
```

Comment:
This plugin extracts all CSS that was generated when processing .css/.less files.  There should no longer be any CSS appearing in the math-input.js bundle.